riak/tests/test_timeseries.py

```diff
@@ -25,21 +27,36 @@
 bd1 = 'временные ряды'
 
 fiveMins = datetime.timedelta(0, 300)
-ts0 = datetime.datetime(2015, 1, 1, 12, 0, 0)
+# NB: last arg is microseconds, 987ms expressed
+ts0 = datetime.datetime(2015, 1, 1, 12, 0, 0, 987000)
```

Comment:
@lukebakken Just wondering.  Could we find a different test case which might not have this rounding problem?